main/parse.c

```diff
@@ -54,6 +55,8 @@ typedef struct {
 	enum specType specType;
 }  parserCandidate;
 
+static ptrArray *parsersUsedInCurrentInput;
```

Comment:
as the initial value is used, please initialize explicitly to `NULL` so it's clear what happens.